pages/services/marathon-lb/mlb-auth/index.md

```diff
@@ -36,11 +36,15 @@ To set up a service account for Marathon-LB, complete the following steps.
 1. [Provision the service account with the necessary permissions.](#give-perms)
 1. [Create a config.json file.](#create-json)
 
-**Requirement:** In `strict` mode, the name of the service account must match the name that the service uses as its `principal`. By default, Marathon-LB uses `mlb-principal` as the name of its `principal`. That's the value that we use in the following procedures. Should you modify the default, you must change `mlb-principal` throughout to match.
+**Note:** This document assumes the following:
+* `marathon-lb` as the marathon-lb marathon service name; if you are installing marathon-lb in a different location, you will have to change the secret location accordingly.
+* `marathon-lb/service-account-secret` as the the full path for the secret used to store the credentials for the Marathon-LB service account; if you change the marathon-lb service name, you will have to change this.
+* `mlb-private-key.pem` as the name of the file containing the private key.
+* `mlb-public-key.pem` as the name of the file containing the public key. 
```

Comment:
the pems  are created as part of the instructions...  they don't seem assumed... though it doesn't hurt having them here.